### PR TITLE
Dragon gibbing fix, host sword checks more

### LINKED ISSUE #17648: Ash drake messages are bugged
# Devour message

> The lesser ash drake devours !

probably should have the name of the person being devoured.

### LINKED ISSUE #17810: Dragons do not regain health from eating corpses
I demand a refund on my dragon, kor.

### PR DESCRIPTION
**Pull request by coiax**
- Drakes now gib() their feast target after printing a message (fixes #17648)
- Drakes qdel() their internal GPS on Destroy()
- Ghost sword now checks for orbiting ghosts approximately every second
- Also probably fixes  #17810

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Why does ghost sword need to check every tick?

**Comment by coiax** [comment]
Because then you can tell when a ghost starts orbiting you, rather than when you're stabbed or doing the stabbing.

**Comment by coiax** [comment]
@KorPhaeron it's your call if you want the new behaviour or the old behaviour. Declare it and it shall be done.

**Comment by KorPhaeron** [comment]
The new behaviour is probably fine, but I'm worried about metaing which ghosts are around you (I guess that could happen in the old system as well though)

File: code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm [hunk 1]
```diff
@@ -30,13 +30,18 @@
...
 	var/swoop_cooldown = 0
-	deathmessage = "collapes into a pile of bones, it's flesh sloughing away."
+	deathmessage = "collapses into a pile of bones, its flesh sloughing away."
 	death_sound = 'sound/magic/demon_dies.ogg'
 	damage_coeff = list(BRUTE = 1, BURN = 0.5, TOX = 1, CLONE = 1, STAMINA = 0, OXY = 1)
 
 /mob/living/simple_animal/hostile/megafauna/dragon/New()
 	..()
 	internal = new/obj/item/device/gps/internal/dragon(src)
+	deathmessage = "[src] [deathmessage]"
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm]: Why is this line needed? Normal death message should work, shouldnt it? Either that or all death messages dont work properly.

Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm]: A reference can't be made before runtime.

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm]: I was more wondering why it wasn't done like this https://github.com/tgstation/-tg-station/pull/17761
